### PR TITLE
Docs 

### PR DESCRIPTION
**Pull request by Developers-codz**
Fixed Indentation and title 
refactor button classes
refactor input button class
fix code snippet indentation and removed utility class from it
added loading lazy in images to optimize image loading

Please do review and give your valuable feedback. Thank you :)

### CODE REVIEW

File: css/style/toast.css [hunk 1]
```diff
@@ -1,41 +1,65 @@
...
-    z-index: 6;
-    left: 50%;
+  visibility: hidden;
+  min-width: 15.625rem;
+  margin-left: -7.81rem;
+  background-color: var(--text-color);
+  color: var(--text-light);
+  text-align: center;
+  border-radius: 2px;
+  padding: 16px;
```
Comment (arjun-computer-geek) [inline css/style/toast.css:9]: It could be better if you will use em instead px for margin and padding.😁

**Review by arjun-computer-geek — APPROVED** [review]
And all rest are good. You can for merge.😊

File: css/style/button.css [hunk 2]
```diff
@@ -1,28 +1,69 @@
...
+  cursor: pointer;
 }
-.reset{
-    cursor:pointer;
+.btn-primary {
+  color: var(--text-light);
+  background-color: var(--primary-color);
+  font-size: 1.2rem;
+  border: none;
+  padding: 0.5rem 1rem;
```
Comment (b0llu) [inline css/style/button.css:10]: if value is less than 1rem you can use px (check on line 13 22 also)

**Review by b0llu — APPROVED** [review]
Everything else is looking good, try to make suggested changes

File: css/style/toast.css [hunk 3]
```diff
@@ -1,41 +1,65 @@
 #toast {
-    visibility: hidden;
-    min-width: 15.625rem;
```
Comment (b0llu) [inline css/style/toast.css:3]: you can just use 15.6 rem instead of .625

File: css/style/toast.css [hunk 4]
```diff
@@ -1,41 +1,65 @@
...
-    background-color: var(--text-color);
-    color: var(--text-light);
-    text-align: center;
-    border-radius: 2px;
-    padding: 16px;
-    position: fixed;
-    z-index: 6;
-    left: 50%;
+  visibility: hidden;
+  min-width: 15.625rem;
```
Comment (b0llu) [inline css/style/toast.css:3]: you can just use 15.6 rem instead of .625

File: css/style/card.css [hunk 5]
```diff
@@ -1,92 +1,90 @@
...
+  min-height: 150px;
+}
+.horizontal-card {
+  max-width: 320px;
+  min-height: 180px;
+}
+.card-img {
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  width: 300px;
```
Comment (Anurag021) [inline css/style/card.css]: It would be better if you can use rem instead of Px rest everything looks good to me.

**Review by Anurag021 — APPROVED** [review]

File: css/style/card.css (hunk 5 above)
Comment (Developers-codz) [reply css/style/card.css]: Thanks but Vikram Sir has said that we can use Px if its value is less than 20

File: css/style/toast.css (hunk 4 above)
Comment (Developers-codz) [reply css/style/toast.css:3]: Yeah, actually I converted px into rem and wrote exact value

File: css/style/toast.css (hunk 1 above)
Comment (Developers-codz) [reply css/style/toast.css:9]: will look into that

**Comment by Developers-codz** [comment]
> You can login to the carbon it will give you short link after that

Then lots of code editing takes place so as of now leaving it like this but will do it in later projects. Thank you.

**Comment by Anurag021** [comment]
yeah.. I understand, no worries....

**Comment by Developers-codz** [comment]
> yeah.. I understand, no worries....

Changes Done Thank you :)

**Pull request merged**